OrchardCore.Commerce/Services/MoneyService.cs

```diff
@@ -1,65 +1,58 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using Microsoft.Extensions.Options;
 using Money;
 using Money.Abstractions;
 using OrchardCore.Commerce.Abstractions;
 using OrchardCore.Commerce.Settings;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace OrchardCore.Commerce.Services
+namespace OrchardCore.Commerce.Services;
+
+/// <summary>
+/// Exposes the data from all enabled currency providers under a simpler, unified interface.
+/// </summary>
+public class MoneyService : IMoneyService
 {
-    /// <summary>
-    /// Exposes the data from all enabled currency providers under a simpler, unified interface.
-    /// </summary>
-    public class MoneyService : IMoneyService
+    private readonly IEnumerable<ICurrencyProvider> _currencyProviders;
+    private readonly CommerceSettings _options;
+    private readonly ICurrencySelector _currencySelector;
+
+    public MoneyService(
+        IEnumerable<ICurrencyProvider> currencyProviders,
+        IOptions<CommerceSettings> options,
+        ICurrencySelector currencySelector)
     {
-        private readonly IEnumerable<ICurrencyProvider> _currencyProviders;
-        private readonly CommerceSettings _options;
-        private readonly ICurrencySelector _currencySelector;
-
-        public MoneyService(
-            IEnumerable<ICurrencyProvider> currencyProviders,
-            IOptions<CommerceSettings> options,
-            ICurrencySelector currencySelector)
-        {
-            _currencyProviders = currencyProviders ?? Array.Empty<ICurrencyProvider>();
-            _options = options?.Value;
-            _currencySelector = currencySelector;
-        }
+        _currencyProviders = currencyProviders ?? Array.Empty<ICurrencyProvider>();
+        _options = options?.Value;
+        _currencySelector = currencySelector;
+    }
 
-        public IEnumerable<ICurrency> Currencies
-            => _currencyProviders
-                .SelectMany(p => p.Currencies)
-                .OrderBy(c => c.CurrencyIsoCode);
+    public IEnumerable<ICurrency> Currencies =>
+        _currencyProviders
+            .SelectMany(p => p.Currencies)
+            .OrderBy(c => c.CurrencyIsoCode);
 
-        public ICurrency DefaultCurrency
+    public ICurrency DefaultCurrency
+    {
+        get
         {
-            get
-            {
-                var defaultIsoCode = _options?.DefaultCurrency;
-                return string.IsNullOrEmpty(defaultIsoCode)
-                    ? Currency.USDollar
-                    : GetCurrency(_options.DefaultCurrency)
-                    ?? Currency.USDollar;
-            }
+            var defaultIsoCode = _options?.DefaultCurrency;
+            return string.IsNullOrEmpty(defaultIsoCode)
+                ? Currency.UsDollar
+                : GetCurrency(_options.DefaultCurrency)
```

Comment:
Use `defaultIsoCode` here, too:
```suggestion
                : GetCurrency(defaultIsoCode)
```